en/orm/associations.rst

```diff
@@ -357,6 +357,23 @@ syntax::
         }
     }
 
+Sometimes you may want to configure composite keys in your associations::
+
+    $this->hasMany('Reviews', [
+        'foreignKey' => [
+            'article_id',
+            'article_name'
+        ],
+        'bindingKey' => [
+            'article_id',
+            'article_name'
+        ]
+    ]);
+
```

Comment:
Sure. I think the binding keys were not well exemplified in this case. I have send a new commit to make it more clearly, but I will also create a new branch in @cakephp/docs to allow new revisions and updates if necessary.